packages/upgrade/src/upgrade.ts

```diff
@@ -1,11 +1,5 @@
 import { dirname, resolve } from 'path';
-import {
-  DiagnosticFixPlugin,
-  EmptyLinesPreservePlugin,
-  EmptyLinesRestorePlugin,
-  LintPlugin,
-  ReRehearsePlugin,
-} from '@rehearsal/plugins';
+import { DiagnosticFixPlugin, LintPlugin, ReRehearsePlugin } from '@rehearsal/plugins';
```

Comment:
We don't need EmptyLinesPreservePlugin and EmptyLinesRestorePlugin anymore! Will be cleaned up after more tests.